### DISCUSSION TITLE
Wi-Fi regular disconnect - Solution

### DISCUSSION THREAD

**Discussion by SimonFili** [arendst/Tasmota#19839] [General]
Hi all,

I've been struggling with all my Tasmota devices disconnecting regularly, like 12 times per day and reconnecting after a few second to 2 mins.

I've finally found the issue.  Maybe is related to specific router vendors, but hopefully, this can help others.
In my case, I have a TP-LINK EAP245 AP.  I've seen this issue on ASUS routers also.

On my network, I have one main SSID with 2.4G and 5G band
I also have a few tests 2.4G SSID

When Tasmota is connected to a SSID with both 2.4G and 5G band, I see those regular disconnect
When Tasmota disconnect from the SSID, devices on the same SSID on the 5G band, are not disconnected.

**So, if you have a disconnect issue, try using an SSID with only a 2.4G band.**

![image](https://github.com/arendst/Tasmota/assets/7669537/0cb98b1d-4fbf-45a7-81e5-1b0708ee8080)

**Comment by sheminasalam** [comment]
did you find any solution? i am having the same issue with my TP link AX1500 router

**Reply by SimonFili** [reply]
Yes, the title of this post is - Solution - 
Have you tried on an SSID with only 2.4G ?
